buildbot/master/files/config/environments.py

```diff
@@ -31,6 +31,7 @@ def __add__(self, other):
 
 build_windows = build_common + Environment({
     'CARGO_HOME': '/home/Administrator/.cargo',
+    'HOME': r'C:\buildbot\slave\windows\build',
```

Comment:
This seems like an odd choice of `$HOME`. Why do we need to set it, and why this value?